### PR TITLE
Feat/accessibility page

### PR DESCRIPTION
**Pull request by davidsauntson**
Adds the accessibility statement page to the design system docs.

### CODE REVIEW

File: design-system-docs/src/_pages/accessibility-statement.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+The first step on this journey was to create a design system that will help us offer a consistent and accessible experience to all, including clients, advisers, volunteers and all staff members.
+
+## What standards do we target?
+
+The Citizens Advice Design System has been designed, built and tested to meet the WCAG 2.1 AA standards.
+
+## Where is the design system used?
+
+We have decided to introduce the design system to the Citizens Advice public website and AdviserNet gradually, making sure there are no disruptions to our services. So far the ‘Immigration’ and ‘Family’ sections use the design system. It is also used by our Intranet, smart meter checking tool, and referrals products.
```
Comment (davidsauntson) [inline design-system-docs/src/_pages/accessibility-statement.md:20]: We might want to change this so we don't have to keep coming back and updating the parts of the public advice site that are migrated to content-platform?
```suggestion
We have decided to introduce the design system to the Citizens Advice public website and AdviserNet gradually, making sure there are no disruptions to our services. So far, some sections use the design system. It is also used by our Intranet, smart meter checking tool, and referrals products.
```

File: design-system-docs/src/_pages/accessibility-statement.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+- The Referrals product was tested on 15th March 2022
+- The Online Scams Tool was tested on 30th June 2022
+
+All tests were carried out by Digital Accessibility Centre (DAC). Tests were carried out in the following conditions:
+
+- Screen sizes: Desktop, Tablet and Mobile
+- Operating systems: Windows, Android and iOS.
+- Browsers: IE11, Firefox, Chrome, Safari, Android browser,
+- Assistive technologies:
+  - JAWS 18 and 2019
```
Comment (marianayovcheva) [inline design-system-docs/src/_pages/accessibility-statement.md:52]: Do we have nested list styles? Not sure if the nested list needs a bit of top spacing but will let Kostas decide

**Review by marianayovcheva — APPROVED** [review]
Nice! ✨

File: design-system-docs/src/_pages/accessibility-statement.md (hunk 2 above)
Comment (davidrapson) [reply design-system-docs/src/_pages/accessibility-statement.md:52]: I'd be inclined to write these as a single-line list. We don't typically encourage nesting lists.

**Review by davidrapson — APPROVED** [review]
👍 This looks great

One suggestion, but non-blocking: I'd be tempted to keep the navigation as just "Accessibility" and title the page "Accessibility and Design System", removing the top-part. Thinking about it a bit more I'm not sure I'd consider this an accessibility _statement_ per-se. It's more a primer on accessibility within the design system.

We should probably reserve the statement for something about the website itself. Maybe I'm over thinking this?

**Pull request merged**